js/ad_manager_vast.js

```diff
@@ -83,22 +256,89 @@ OO.Ads.manager(function(_, $) {
       'close' ];
 
     /**
-     * Helper function to verify that the xml has a valid vast ad in it and that it is a valid xml.
+     * Helper function to verify that XML is valid
      * @param {xml} vastXML Contains the vast ad data to be parsed.
```

Comment:
can a @public tot he comments if making this public or @private if not, why is this public?